### PR TITLE
Explicitly require utf-8 encoding on all files

### LINKED ISSUE #915: Syntax Error when import nifgen.enums
* Description of issue

import nifgen.enums

errors with a SyntaxError

import nifgen.enums
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\workspaces\jdengel-PXI-py27_64bit_prototype\Python27\lib\site-package
s\nifgen\__init__.py", line 4, in <module>
    from nifgen.enums import *          # noqa: F403,F401,H303
  File "C:\workspaces\jdengel-PXI-py27_64bit_prototype\Python27\lib\site-package
s\nifgen\enums.py", line 64
SyntaxError: Non-ASCII character '\xe2' in file C:\workspaces\jdengel-PXI-py27_6
4bit_prototype\Python27\lib\site-packages\nifgen\enums.py on line 65, but no enc
oding declared; see http://python.org/dev/peps/pep-0263/ for details

* `python -c "import niscope; niscope.print_diagnostic_information()"` output (replace `niscope` with appropriate package name)

    ```
OS:
    Version:   6.1.7601
    Bits:      64
    Name:      Windows
Driver:
    Version:   Unknown
    Name:      NI-SCOPE
Module:
    Version:   1.0.0
    Name:      niscope
Python:
    Version:   2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:24:40) [MSC v.1500
64 bit (AMD64)]
    Bits:      64
    Is_Venv:   False
    Installed Packages:
        alabaster==0.7.9
        appdirs==1.4.0
        argparse==1.2.1
        astroid==1.0.1
        babel==2.3.4
        backports.ssl-match-hostname==3.4.0.2
        bitstring==3.1.5
        colorama==0.3.9
        coverage==3.7.1
        cov-core==1.15.0
        crcmod==1.7
        cycler==0.10.0
        decorator==4.1.2
        docutils==0.11
        dominate==2.3.1
        enum34==1.1.6
        et-xmlfile==1.0.1
        ez-setup==0.9
        functools32==3.2.3.post2
        glob2==0.6
        imagesize==0.7.1
        jdcal==1.2
        jinja2==2.7.2
        lazy-object-proxy==1.3.1
        logilab-common==0.61.0
        lxml==3.4.0
        markupsafe==0.18
        matplotlib==2.0.2
        mock==1.0.1
        networkx==2.0
        nidaqmx==0.5.7
        nidcpower==1.0.0
        nidmm==1.0.0
        nifgen==1.0.0
        nimodinst==1.0.0
        niscope==1.0.0
        node==0.9.12
        numpy==1.14.5+mkl
        odict==1.5.1
        olefile==0.44
        openpyxl==2.3.2
        pandas==0.23.3
        pillow==4.2.0
        pip==9.0.1
        pluggy==0.4.0
        plumber==1.2
        py==1.4.30
        pycodestyle==2.3.1
        pygments==2.2.0
        pylint==1.1.0
        pyparsing==2.0.1
        pypiwin32==219
        pyreadline==2.0
        pyserial==2.7
        pyside==1.2.4
        pytest==2.7.2
        pytest-capturelog==0.7
        pytest-cov==1.6
        pytest-quickcheck==0.8.3
        pythonnet==2.3.0
        python-dateutil==2.2
        python-docx==0.8.5a0
        pytz==2017.2
        pyvisa==1.9.0
        pyyaml==3.12
        requests==2.7.0
        scipy==1.1.0
        setuptools==18.0.1
        singledispatch==3.4.0.3
        six==1.5.2
        snowballstemmer==1.2.1
        sphinx==1.4.8
        sphinx-rtd-theme==0.1.9
        tornado==3.2
        tox==2.6.0
        typing==3.6.2
        unidecode==1.0.22
        unum==4.1.3
        virtualenv==14.0.6
        wrapt==1.10.10
        wxpython==4.0.1
        xlrd==0.9.3
        xlsxwriter==0.7.7
        xlwt==1.0.0
        yapsy==1.11.223
        zope.component==4.2.0
        zope.deprecation==4.1.0
        zope.event==4.0.2
        zope.interface==4.1.0
        zope.lifecycleevent==4.0.3
        uli==1.18.11.dev-r0
        usbio==0.1.dev-r0
    ```

* Steps to reproduce issue

    ``` python
    import nifgen.enums
    ```

if we place 

# * coding: utf-8

as defined in the PEP a the top of the file then this resolves the issue that we see.

### PR DESCRIPTION
**Pull request by texasaggie97**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] ~~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~
- [X] ~~I've added tests applicable for this pull request~~
### What does this Pull Request accomplish?
* Add `# -*- coding: utf-8 -*-` to the top of all files
* Explicitly tell Python interpreter we need utf-8 encoding (per [PEP 263](https://www.python.org/dev/peps/pep-0263/))
* Added to all files whether we need utf-8 at this point or not
### List issues fixed by this Pull Request below, if any.
* Fix #915 

### What testing has been done?
* Unit
* System

### CODE REVIEW

File: build/templates/_converters.py.mako [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<%
+# Have to put this in a variable and add it that way because mako keeps thinking it is for it, not for the output file
```
Comment (marcoskirsch) [inline build/templates/_converters.py.mako]: I think you can [do this](https://stackoverflow.com/a/30750520/316875):
```
## -*- coding: utf-8 -*-
```

File: build/templates/errors.py.mako [hunk 2]
```diff
@@ -1,4 +1,9 @@
+<%
+# Have to put this in a variable and add it that way because mako keeps thinking it is for it, not for the output file
+encoding_tag = '# -*- coding: utf-8 -*-'
+%>\
 #!/usr/bin/python
```
Comment (marcoskirsch) [inline build/templates/errors.py.mako]: We're inconsistent about this, some files have it, some don't. Isn't this useful only for scripts that are meant to be executed from the command line?

File: build/templates/_converters.py.mako (hunk 1 above)
Comment (texasaggie97) [reply build/templates/_converters.py.mako]: Then mako assumes it is for it and removes it and it ends up not being in the generated file

Comment (marcoskirsch) [reply build/templates/_converters.py.mako]: Did you try `##` instead of `#`?

Comment (texasaggie97) [reply build/templates/_converters.py.mako]: Yes.

**Comment by texasaggie97** [comment]
I did try `##` and it didn't matter.
Moved to template_params and used from there.

**Pull request merged**